recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu.cfg

```diff
@@ -5,7 +5,7 @@
 # This is a fairly minimal example of what is required for a
 # Paravirtualised Linux guest. For a more complete guide see xl.cfg(5)
 
-disk = [ 'phy:/dev/sda1,xvda1', 'phy:/dev/mmcblk1p1,xvda2' ]
+#disk = [ 'phy:/dev/sda1,xvda1', 'phy:/dev/mmcblk1p1,xvda2' ]
```

Comment:
Is that change belongs to vGPU support?

Reply:
no, will remove

Reply:
well, is this a part of the reverted file?

Reply:
all unrelated changes (memory + disk) will be reverted

Reply:
Sorry, I associated this change to a wrong patch.

+1